cmd/zed/agents/zfs_retire.c

```diff
@@ -71,8 +71,7 @@ zfs_retire_clear_data(fmd_hdl_t *hdl, zfs_retire_data_t *zdp)
  */
 typedef struct find_cbdata {
 	uint64_t	cb_guid;
-	const char	*cb_fru;
-	zpool_handle_t	*cb_zhp;
+	zpool_handle_t  *cb_zhp;
```

Comment:
nit: it looks like there's a gratuitous whitespace change here.  This line should be unchanged.